lib/models/addons-factory.js

```diff
@@ -59,7 +59,7 @@ class AddonsFactory {
         } catch (e) {
           project.ui.writeError(e);
           const SilentError = require('silent-error');
-          throw new SilentError(`An error occured in the constructor for ${addonInfo.name} at ${addonInfo.path}`);
+          throw new SilentError(`An error occurred in the constructor for ${addonInfo.name} at ${addonInfo.path}`);
```

Comment:
that's correct but those error messages are certainly not public API, so I don't consider this an actual issue :)